edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -26,7 +26,7 @@
       {% endblocktrans %}
     </p>
     <p>
-      <span><a href="{{software_download_url}}" target="_blank">Start System Check</a></span>
+      <span><a href="{{software_download_url}}" id="software_download_link" data-action="click_download_software" target="_blank">Start System Check</a></span>
```

Comment:
Remove the href from here and redirect to the URL once the ajax call returns successfully.